app/src/main/java/com/google/android/samples/socialite/ui/Bubble.kt

```diff
@@ -16,25 +16,37 @@
 
 package com.google.android.samples.socialite.ui
 
+import androidx.compose.animation.AnimatedContent
+import androidx.compose.animation.AnimatedContentScope
+import androidx.compose.animation.ExperimentalSharedTransitionApi
+import androidx.compose.animation.SharedTransitionLayout
+import androidx.compose.animation.SharedTransitionScope
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import com.google.android.samples.socialite.ui.chat.ChatScreen
 
+@OptIn(ExperimentalSharedTransitionApi::class)
 @Composable
 fun Bubble(chatId: Long) {
-    SocialTheme {
-        ChatScreen(
-            chatId = chatId,
-            foreground = false,
-            onBackPressed = null,
-            // TODO (donovanfm): Hook up camera button in the Bubble composable
-            onCameraClick = {},
-            // TODO (jolandaverhoef): Hook up play video button in the Bubble composable
-            onVideoClick = {},
-            // TODO (mayurikhin): Hook up camera button in the Bubble composable
-            onPhotoPickerClick = {},
-            modifier = Modifier.fillMaxSize(),
-        )
+    SharedTransitionLayout {
+        AnimatedContent(targetState = 1) { _ ->
```

Comment:
Is this a preview? Or where is this used? 
If shared elements aren't required in these examples, you could consider creating a `Modifier.trySharedElement()` that doesn't nessecarily need the scopes and just does a no-op when scopes are not present, and adds the modifiers when scopes are present. Then in screens where its not used, you wouldn't provide the scopes.

Reply:
Otherwise I'd change this to `AnimatedVisibility(true)` for a bit of a cleaner description.